lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -107,6 +107,7 @@ export default class BitcoinProcessor {
     this.quantileCalculator = new SlidingWindowQuantileCalculator(BitcoinClient.convertBtcToSatoshis(1),
       ProtocolParameters.windowSizeInGroups,
       ProtocolParameters.quantileMeasure,
+      ProtocolParameters.quantileDeviationWindowFactor,
```

Comment:
Would change factor to percentage be easier to comprehend? e.g.
`maxQuantileDeviationPercentage`, or `maxQuantileChangePercentage`, or `maxSubsequentQuantileChangePercentage`?

Reply:
I was thinking about that too but wanted to stick with Arvind's original suggestions (and also not do the math :) ...) but I agree and will update.